README.md

```diff
@@ -0,0 +1,28 @@
+<h2>dev-ace: An AI Engine Based on LLaMA</h1>
+
+<p>
+    This repository, is a branch of the 
+    <a href="https://github.com/conormckenzie/reasoning-engine">reasoning-engine</a> project, 
+    focusing on developing an AI engine based on LLaMA.
+</p>
+
+<h2>Overview</h2>
+<p>
+    The branch will be used to push and pull to and from the main branch
+</p>
```

Comment:
Overview can stay perhaps, but it should be an overview of the project, not of the branch's purpose.

That said, it can make sense to have the readme of a feature branch explain the feature or changes introduced in it, but in this case the readme should not be merged into the dev branch.